src/a.tex

```diff
@@ -285,7 +348,7 @@ \section{Eventual Success of Store-Conditional Instructions}
 cache and TLB size and associativity.
 Similarly, we disallowed other loads and stores within the loops to avoid
 restrictions on data-cache associativity in simple implementations that track
-the reservation within the cache.
+the reservation within a non-shared cache.
```

Comment:
It's not impossible to do this with a shared cache, just more difficult.

Reply:
Okay.  Then it's no longer a simple implementation, right?  The concern is about accidentally leading implementors in a non-productive direction.  These are subtle issues that can easily not be caught until after release, or still cause waste of effort if caught in time.

Reply:
That's a fair point.